recipes/openfe/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "openfe" %}
+{% set version = "0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/OpenFreeEnergy/openfe/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 03b6531ed7c8b03f09294c9792ce465633c1d2e6aa60d0e34344a14b3dc56b96
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+  run:
+    - click
+    - coverage
+    - lomap2
+    - networkx
+    - numpy
+    - openff-toolkit
+    - pip
+    - plugcli
+    - pytest
+    - pytest-cov
+    - pytest-xdist
+    - python >=3.8
+    - rdkit
+    - typing-extensions
```

Comment:
Most of these are not upstream, are you sure?

Ref: https://github.com/OpenFreeEnergy/openfe/blob/03cf8a1ca3db5af90a6220742dc2d105173cc0e6/setup.cfg#L24-L28

Reply:
Ahh ok – perfect!